components/image/image.css

```diff
@@ -0,0 +1,32 @@
+.round-img-xl, .round-img-lg{
+    margin:2rem;
+}
+.round{
+    border-radius:50% ;
+}
+.round-img-xl{
+    height:35rem;
```

Comment:
try to keep height mostly auto to make it responsive with respect to the container